coalib/settings/ConfigurationGathering.py

```diff
@@ -179,6 +179,10 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    if "EDITOR" in os.environ and "editor" not in sections["default"]:
+        sections["default"].add_or_create_setting(
+            Setting("editor", os.environ["EDITOR"]))
```

Comment:
I think this is the wrong place to do this in, rather do it right in the OpenEditorAction? The EDITOR shouldn't be written to the sections as we may also decide to store it back to file later